example/transaction/customGiftcard.js

```diff
@@ -0,0 +1,25 @@
+import buckarooClient from '../buckarooClient';
+
+const giftcard = buckarooClient.method('giftcard');
+
+//Pay
+giftcard
+    .pay({
+        amountDebit: 10,
+        description: 'Custom Giftcard Payment',
+        name: 'customgiftcard3',
+        cardNumber: '0000000000000000001',
+        pin: '1000',
+    })
+    .request();
+//Refund
+giftcard
+    .refund({
+        originalTransactionKey: 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
+        amountCredit: 10,
+        invoice: 'Custom Giftcard Refund',
```

Comment:
can you fix this invoice value to not confuse anyone, as we did in afterpay
`invoice: uniqid()`

Reply:
Fixed